FeatureTest.java

```diff
@@ -6,7 +6,7 @@ public static void main(String[] args) {
     public int square(int variable){
         String str = null;
```

Comment:
[code-reviewer[bot]]
Method parameter 'args' might be null - consider adding null check